components/tab/TitleTab.js

```diff
@@ -0,0 +1,43 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../box';
+import cx from 'classnames';
+import theme from './theme.css';
+
+import { Heading4 } from '../typography';
+
+class TitleTab extends PureComponent {
+  static propTypes = {
+    active: PropTypes.bool,
+    children: PropTypes.node,
+    className: PropTypes.string,
+    counter: PropTypes.node,
+    element: PropTypes.node,
+  };
+
+  static defaultProps = {
+    element: 'a',
+    active: false,
+  };
+
+  render() {
+    const { active, children, className, counter = null, ...others } = this.props;
+    const classNames = cx(theme['title-tab'], { [theme['is-active']]: active }, className);
+
+    return (
+      <Box
+        data-teamleader-ui="title-tab"
+        className={classNames}
+        marginHorizontal={3}
+        paddingHorizontal={3}
+        paddingVertical={4}
+        {...others}
+      >
+        <Heading4>{children}</Heading4>
```

Comment:
Using a `h4` isn't very semantically correct in this case. Luckily the `Heading4` component has a prop to change the element 😉  I would use a span instead.

Reply:
You're right 👍 

 :ok: by @InstaK on 29/11/2017 16:36:04